Automattic/wp-calypso
# Plans Next: Fix misaligned titles in comparison grid

Fixes https://github.com/Automattic/martech/issues/2389

## Proposed Changes

Fixes the plan titles in comparison from being misaligned due to select dropdown shown/hidden

## Media

### Before

<img width="500" alt="Screenshot 2024-03-26 at 2 31 15 PM" src="https://github.com/Automattic/wp-calypso/assets/1705499/160dd767-f009-4e35-a37e-93668db6d12e">

### After

<img width="500" alt="Screenshot 2024-03-26 at 2 34 38 PM" src="https://github.com/Automattic/wp-calypso/assets/1705499/960b3ab6-067c-46a1-a5fa-903856c07415">

## Testing Instructions



* Go to `/plans/[ site ]`
* Open the comparison grid and confirm the media above
    * may need to narrow/resize viewport to activate the selectors (chevrons)
    * make sure one of the columns doesn't have a plan with a selector (chevron) shown next to the plan name

## Pre-merge Checklist
